.buildlibrary

```diff
@@ -1,4 +1,4 @@
-ValidationKey: '613522360'
+ValidationKey: '6.14e+08'
```

Comment:
never seen scientific notation here :D

Reply:
:rofl:

Reply:
Turns out, today is a good day for that to happen, and version numbers ending on the digit 0 or five help too.